[IMP] POS: homepage update (#13761)
odoo/documentation: content/applications/sales/point_of_sale.rst

Point of Sale
=============

Odoo **Point of Sale** is designed for managing shops and restaurants. It is web-browser-based,
allowing it to run on any device, and is built to maintain functionality even during temporary
network outages.

Beyond traditional :doc:`stores <point_of_sale/shop>` and :doc:`restaurants
<point_of_sale/restaurant>` settings, Odoo :abbr:`POS (Point of Sale)` also supports a
:doc:`self-ordering <point_of_sale/self_order>` feature, enabling customers to place orders and make
payments using a dedicated kiosk or their own mobile devices.

As with all Odoo applications, Odoo Point of Sale is fully integrated with the entire Odoo
ecosystem. This integration primarily ensures that:

- Product movements are automatically tracked and registered in the :doc:`Inventory
  <../inventory_and_mrp/inventory>` application.
- All POS transactions are recorded in the :doc:`Invoicing <../finance/accounting>` or, if
  installed, :doc:`Accounting <../finance/accounting>` application.
- :doc:`Sales orders <point_of_sale/shop/sales_order>` created from the :doc:`Sales <sales>` or

>   :doc:`Website <../websites>` application can be retrieved and settled in Point of Sale.

Reviewer: I'm not 100% sure this is worth mentioning here: is there a specific use case where this workflow would make sense?

.. _pos/open-register:

Use
===

Once the POS is fully :doc:`configured <point_of_sale/configuration>`, access the POS interface by
opening the register. Navigate to :menuselection:`Point of Sale --> Dashboard` and:

#. On a POS card, click :guilabel:`Open Register`.
#. At the :guilabel:`Opening Control` screen, ensure the :guilabel:`Opening cash` amount is correct.
#. Click :guilabel:`Open Register` again.

.. note::
   - Once the register is open, :guilabel:`Open Register` is replaced by :guilabel:`Continue
     Selling` on the POS card.
   - :doc:`Multiple users <point_of_sale/employee_login>` can be logged into the same register at
     the same time. However, the register can only be opened once on the same browser.

From the top header of the POS interface:

- Click :guilabel:`Register` to access the register for daily POS actions such as :ref:`sales
  <pos/sell>`, :ref:`refunds <pos/refund>`, etc.
- Click :guilabel:`Orders` to access the POS :ref:`orders <pos/orders>` overview screen to retrieve
  past or ongoing orders.
- Click the :icon:`fa-plus-circle` :guilabel:`(plus)` icon to put the current order aside and start
  a new one.

> - Click the order numbers to jump between ongoing orders.

Reviewer: ```suggestion
- Click the order numbers to switch between ongoing orders.
```
or navigate

- Search for products using the search bar.
- Click the :icon:`fa-barcode` (:guilabel:`barcode`) icon to use a webcam as a barcode scanner.

> - Click the user's avatar to :ref:`switch between employees <pos/employee_login/use>`.

Reviewer: ```suggestion
- Click the user's avatar to switch between employees, provided :ref:`multi-employee management is enabled <pos/employee_login/use>`.
```
I think it's worth it clarify that this isn't done by default? (for example I tried clicking the avatar and nothing happened but then I used my brain and checked the referenced doc)

- Click the :icon:`fa-bars` (:guilabel:`hamburger menu`) icon to access more advanced options, such
  as :ref:`closing the register <pos/register-close>`.

.. _pos/orders:

> Orders overview

Reviewer: I would move this section under *Return and refund products*: I think it would make more sense workflow-wise (you don't have any orders unless you *Sell products* and then since *Set customers* and *Return and refund products* actually relate closely to selling products (and creating orders), maybe it'd be best to keep them close together.

---------------

The :guilabel:`Orders` overview allows for viewing, searching, and retrieving orders from the POS
interface. Access this view by clicking :guilabel:`Orders` on the interface top header.

Then, search for orders in the search bar using their:

- :guilabel:`Reference`
- :guilabel:`Receipt Number`
- :guilabel:`Date`
- :guilabel:`Customer`
- :guilabel:`Delivery Channel`
- :guilabel:`Delivery Order Status`

To filter orders based on their status, click the :guilabel:`Active` dropdown menu and select one of
the following options:

> - :guilabel:`Active`: Orders in progress, either marked as :guilabel:`Ongoing`, in the

Reviewer: next

      keypad.
#. Once an order is completed, proceed to checkout by clicking :guilabel:`Payment`.

   #. Select the **payment method**.
   #. Enter the received amount.
   #. Click :guilabel:`Validate`.
#. Finally, click :guilabel:`New Order` to move on to the next customer.

.. image:: point_of_sale/pos-register.png
   :alt: POS register
   :scale: 45 %

.. tip::
   - Use both `,` and `.` on the keyboard as decimal separators.
   - **Cash** is selected by default if no payment method is manually selected.

.. _pos/customers:

Set customers
-------------

Registering customers is necessary to :doc:`collect their loyalty points and grant them rewards
<point_of_sale/pricing/loyalty>`, automatically apply an :doc:`attributed pricelist
<point_of_sale/pricing/pricelists>`, or :ref:`generate and print invoices
<pos_invoices/invoices>`.

To create customers from an :ref:`the POS register <pos/open-register>`:

#. Click :guilabel:`Customer`.
#. Click :guilabel:`Create`.
#. Then, complete the contact information.

To create customers from the backend:

#. Go to :menuselection:`Point of Sale --> Orders --> Customers`.
#. Click :guilabel:`New`.

> #. Then, fill in the information and save.

Reviewer: same as above

```suggestion
#. Fill in the information and save.
```
(also technically it is saved automatically so no real need to manually save)

     :guilabel:`Date`, or :guilabel:`Customer`, enter a value in the search bar and choose the
     relevant filter from the dropdown menu.
   - When the total amount is negative, adding a gift card to the cart automatically adjusts the
     gift card balance to match that amount.

.. note::
   Alternatively, a refund can be processed by selecting the returned product(s) from the POS
   register and setting a negative quantity equal to the number of returned items. To do so, click
   :guilabel:`Qty` and :guilabel:`+/-`, and update the quantity accordingly.

Once the return payment is validated, Odoo generates the required credit note, referencing the
original receipt or invoice and partially or fully canceling the entry.

.. _pos/customer-notes:

Notes
-----

Customer and general notes
~~~~~~~~~~~~~~~~~~~~~~~~~~

Notes for customers appear on :ref:`receipts <pos/configuration/receipts>` and invoices. These notes
come in two forms: customer notes for specific products and general notes for the entire order. To
add a **customer note** from the :ref:`POS register <pos/open-register>`:

#. Select a product from the cart.

> #. Click :guilabel:`Actions`.

Reviewer: ```suggestion
#. Click the :icon:`fa-vertical-ellipsis` (:guilabel:`vertical ellipsis`) button.
```

#. Select :guilabel:`Customer Note`, which opens a pop-up window to add or modify the note's
   content.

.. note::
   Product notes from an :doc:`imported SO <point_of_sale/shop/sales_order>` are displayed
   identically in the cart.

To add a **general note** for the complete order:

#. Click :guilabel:`Actions`, and
#. :guilabel:`General Note`, which opens a pop-up window. To add or modify the note's content:

   - Type the note directly into the window, or
   - Use a configured note model to save time if the same content is frequently used. Click on the
     desired note model to insert its text.

To set up note models:

#. Navigate to :menuselection:`Configuration --> Note Models`.

> #. Click :guilabel:`New` and enter the desired text for your note model. Then, save it.

Reviewer: This could also be one sentence (especially since saving manually is optional):

```suggestion
To set up note models, navigate to :menuselection:`Configuration --> Note Models`, click :guilabel:`New`, and enter the desired text.
```

Internal notes
~~~~~~~~~~~~~~

In parallel, **internal notes** attach to items similarly but are destined to other users. For
example, an additional information about an order for a cook, such as `no tomato`. That information,
while useful, shouldn't be printed on the customer receipt. To add an **internal note**:

#. Select an item from the cart.
#. Click :guilabel:`Internal Note`, which opens a pop-up window to add or modify the note's content.

> 

Reviewer: I think this sentence could be simplified (both in form and content) and the instructions could be grouped in a single sentence:

```suggestion
Internal notes are similar to customer and general notes but are meant for internal users only and do not appear on the customer receipt (e.g., additional information about an order for the kitchen, such as `no tomato`). To add an **internal note** to an order, select an item from the cart, click :guilabel:`Internal Note`, and add or modify the note's content in the popup.
```
(but then I wonder: what about kitchen notes? what's the difference?)

> .. image:: point_of_sale/customer-notes.png

Reviewer: great screenshot!

   :alt: Customer note button and notes (SO and POS register) on products in the cart
   :scale: 50 %

.. _pos/cash-register:

Manage the cash register
------------------------

Odoo POS allows you to determine which coins and bills are accepted. To set up the allowed coins and
bills:

> #. Navigate to :menuselection:`Configuration --> Coins/Bills`.

Reviewer: ```suggestion
#. Navigate to :menuselection:`Point of Sale --> Configuration --> Coins/Bills`.
```

#. Click :guilabel:`New` to add a new value.
#. Select the POS where this value is available in the :guilabel:`Point of Sale` column.

To record a cash in or cash out transaction not associated with a sale:

#. Click :icon:`fa-bars` (:guilabel:`hamburger menu`) on the POS interface.
#. Click :guilabel:`Cash In/Out`, which opens a pop-up window.
#. Select :guilabel:`Cash In` or :guilabel:`Cash Out`.
#. Enter the amount.
#. Specify the reason for this addition or removal of cash, and :guilabel:`Confirm`.

.. note::
   Some users might not be allowed to do cash in or out actions depending on their :doc:`POS access

>    rights <point_of_sale/employee_login>`.

Reviewer: ```suggestion
   Some users might not be allowed to perform cash in or out actions depending on their :doc:`POS access
   rights <point_of_sale/employee_login>`.
```
EDIT: but actually I think it might more useful to phrase it differently:

```suggestion
   Only employees with :ref:`basic or advanced access rights <pos/employee_login/configuration>` are allowed to perform cash in/out actions.
```

Close the POS register
----------------------

To close the POS register, click :icon:`fa-bars` (:guilabel:`hamburger menu`) and :guilabel:`Close
Register`.

Doing so opens the :guilabel:`Closing Register` pop-up screen. From this screen, you can retrieve:

- The number of orders and the total amount made during the session.
- The expected amounts grouped by payment method.

#. Click :icon:`fa-money` (:guilabel:`money`) to specify the number of each coin and bill.
#. Click :guilabel:`Confirm` to return to the previous pop-up window. The computed amount is set in
   the :guilabel:`Cash Count` field, and the :guilabel:`Closing Details` are specified in the
   :guilabel:`Closing Note` section.
#. Click :guilabel:`Close Register` to close the register and post accounting entries.

.. tip::
   Click :icon:`fa-clone` (:guilabel:`clone`) to automatically fill in the field with the expected
   cash amount.

.. note::
   - To abort, click :guilabel:`Discard` on the pop-up window.
   - When the money counted does **not** match the expected amount, a :guilabel:`Payments
     Difference` window automatically pops up. Click :guilabel:`Proceed Anyway` to log the
     difference for the books.

>    - In restaurant, closing the register when orders are still in draft is not allowed and opens a

Reviewer: ```suggestion
   - Closing the register of a restaurant POS when orders are still in draft is not allowed and opens a
```
i'd also add a ref for restaurant
